src/components/WeatherForecasts/WeatherForecasts.module.scss

```diff
@@ -0,0 +1,59 @@
+@import "assets/styles/variables";
+
+.weatherForecasts {
+  position: relative;
+  display: grid;
+  grid-template-columns: minmax(0, 1fr) 200px;
+}
+
+.weatherForecastsSwiper {
+  display: flex;
+}
+
+.weatherForecastsNavigation {
+  z-index: 100;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-right: 40px;
+  padding-left: 40px;
+}
+
+.weatherForecastsItemWrapper {
+  display: flex;
+  align-items: center;
+}
+
+.weatherForecastsItem {
+  padding-right: 30px;
+  padding-left: 30px;
+}
+
+.weatherForecastsNavigationButton {
+  width: 50px;
+  height: 50px;
+  background-color: rgba($afro, 0.3);
+  border: none;
+  border-radius: 50%;
+  cursor: pointer;
+
+  &::after {
+    display: inline-block;
+    padding: 4px;
```

Comment:
$space-xs

Reply:
we don't use it for paddings cause we would have too many variables for different padding cases